### ISSUE TITLE
Cannot distinguish between stdout and stderr in the `Exec` task

### ISSUE THREAD

**Issue by ltrzesniewski** [dotnet/msbuild#6838]
The `Exec` task can capture the output of the executed command by setting `ConsoleToMsBuild="true"` and reading the `ConsoleOutput` output parameter.

Unfortunately, the standard output and standard error streams are always combined: it's not currently possible to retrieve only the standard output for instance. This is an issue for commands that return the requested data in the standard output, but also write additional messages to the standard error.

One way to solve this would be to add additional output properties such as `StandardOutput`/`StandardError` (and maybe not trim those since the user probably wants the "raw" output when using them).

**Comment by BenVillalobos** [comment]
Team Triage: This is currently unsupported. Have you tried redirecting the stderr of the exec call to some file using `2> stderr.txt`? That should work around this missing feature.

**Comment by ltrzesniewski** [comment]
Yes, I worked around it, but figured I should report the issue in case anyone else needs it.
